modules/configure-cluster-decommissioning.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * configuration/cluster-decommissioning.adoc
+:_content-type: PROCEDURE
+[id="configure-cluster-decommissioning_{context}"]
+= Configuring cluster decommissioning
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *System Configuration*.
+. In the *System Configuration* header, click *Edit*.
+. In the *Cluster deletion* section, you can configure the following fields:
+* *Decommissioned cluster age*: The number of days that a cluster is unreachable before it is considered for deletion. If Central cannot reach Sensor on the cluster for this number of days, the cluster and all of its resources are automatically deleted. To leave this feature disabled, which is the default behavior, enter 0 in this field. To enable this feature, enter a number, such as 90, to configure the number of unreachable days. 
+* *Ignore clusters which have labels*: To prevent a cluster from deletion, you can configure a label using a key and value in this section. Clusters with this label will not be deleted, even if they have been unreachable for the number of days set in the *Decommissioned cluster age* field.
+
+** *Key*: Enter the label to use for the cluster.
+** *Value*: Enter the value associated with the key. 
++
+For example, to preserve production clusters from deletion, you can configure a key of `cluster-type` and a value of `production`.
++
+[NOTE]
+====
+In the *Cluster deletion* section, click *Clusters which have Sensor Status: Unhealthy* to go to the Clusters list page. This page is filtered to show unhealthy clusters that are subject to deletion and a timeframe for the deletion.
+====
```

Comment:
Can we also add to the notes that if the feature is enabled while a cluster has already been unhealthy for several days, the counting of days will start from the time cluster became unhealthy and not from the time when feature is enabled ? And that if there are any unhealthy clusters that you do not want deleted, they should be ignored ?

Reply:
> And that if there are any unhealthy clusters that you do not want deleted, they should be ignored ?

Just checking- do you mean that they should label them using the key/value which will then cause them to be ignored?

Reply:
Yes that's right